### PR TITLE
fix xDAI issues

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1901
Resolves: https://github.com/daostack/alchemy/issues/1904
Resolves: https://github.com/daostack/alchemy/issues/1914
Resolves: https://github.com/daostack/alchemy/issues/1875

* Updates the provider RPC URL
* clones arc.js code for `ethBalance` to poll instead of creating a subscription that uses wss, but only on xDAI.  This is until we can get a provider  RPC URL that supports wss.  Other than xDAI, we use the arc.js code that we always have.
* The review app is pointing to xdai.
* affects:
  * AccountProfilePage (account balance)
  * staking buttons tooltip and whether predictions are enabled in Proposal Card and Details (DAO balance)
  * Redeem button tooltip in Redemptions menu (DAO balance)
  * DAO side bar (DAO balance)
  * Header account dropdown (account balance)

### CODE REVIEW

File: src/components/Account/AccountBalances.tsx [hunk 1]
```diff
@@ -68,7 +69,8 @@ export default withSubscription({
       return of(null);
     }
     const daoState = dao;
-    const arc = daoState.dao.context;
+    const arc = getArc();
```
Comment (orenyodfat) [inline src/components/Account/AccountBalances.tsx:72]: this change is for all networks ? why ?

File: src/settings.ts [hunk 2]
```diff
@@ -200,8 +200,8 @@ export const settings = {
     graphqlHttpProvider: process.env.ARC_GRAPHQLHTTPPROVIDER || SubgraphEndpoints.http_xdai,
     graphqlWsProvider:  process.env.ARC_GRAPHQLWSPROVIDER || SubgraphEndpoints.ws_xdai,
     graphqlSubscribeToQueries: false,
-    web3Provider:  process.env.ARC_WEB3PROVIDER || "wss://poa.api.nodesmith.io/v1/dai/jsonrpc?apiKey=128059b9320a462699aef283a7ae2546",
-    web3ProviderRead:  process.env.ARC_WEB3PROVIDERREAD || "wss://poa.api.nodesmith.io/v1/dai/jsonrpc/ws?apiKey=128059b9320a462699aef283a7ae2546",
+    web3Provider: process.env.ARC_WEB3PROVIDER || "wss://poa.api.nodesmith.io/v1/dai/jsonrpc/ws?apiKey=128059b9320a462699aef283a7ae2546",
```
Comment (orenyodfat) [inline src/settings.ts:203]: nodesmith is not live project .

Comment (dkent600) [reply src/settings.ts:203]: @orenyodfat This is still a draft PR, not yet meant to be reviewed.  As usual, when it is ready for your review, I'll request it via the usual Github process.

File: src/components/Account/AccountBalances.tsx (hunk 1 above)
Comment (dkent600) [reply src/components/Account/AccountBalances.tsx:72]: This is just a minor refactor to bring the code in alignment with how we are supposed to be obtaining the global instance of `Arc`.

**Pull request merged**